src/hooks/users.ts

```diff
@@ -48,7 +130,7 @@ export const useIsVenueUsersLoaded = () => {
   // We want to gradualy move from that approach to a more modular one, where the specific data is connected where it is required
   useConnectVenueUsers();
 
-  const users = useSelector(usersSelector);
+  const users = useSelector(universeUsersSelector);
```

Comment:
Since this is in`useIsVenueUsersLoaded ` it should probably use a selector called `venueUsersSelector`. Alternatively, the name of this hook should be changed to something like `useIsUniverseUsersLoaded `.